recipe/run_test.sh

```diff
@@ -23,9 +25,18 @@ ncks -O --rgr grid=grd_2x2.nc \
         foo.nc
 
 
+echo "ncap2"
```

Comment:
Thanks for organizing our tests :smile:

Reply:
That's just me debugging...

Reply:
You can leave the `echo`s or make them a comment.
It will help us in the future.